pyzeebe/worker/task_router.py

```diff
@@ -71,7 +72,7 @@ def task(
         def task_wrapper(task_function: Callable):
             config = TaskConfig(
                 task_type,
-                exception_handler,
+                exception_handler or self._default_exception_handler,
```

Comment:
Would be nice to add  a small unit test for this

Reply:
I'll add it in #349